libafl/src/stages/sync.rs

```diff
@@ -134,6 +134,9 @@ where
                 .unwrap()
                 .left_to_sync
                 .retain(|p| p != &path);
+            let Some(input) = input else {
+                continue;
```

Comment:
This is dangerous, don't just ignore everything. Especially not without at least logging it.

Reply:
this is not about Ok/Err but Some/None

Reply:
I agree at least a log entry should be emitted.